assets/js/app.js

```diff
@@ -135,10 +135,19 @@ let footer = $(`
               </div>
                 <form>
```

Comment:
The above files are renamed without any requirement leading to error in other sections